examples/chart/teleport-kube-agent/templates/deployment.yaml

```diff
@@ -65,6 +65,10 @@ spec:
     {{- toYaml .Values.extraVolumeMounts | nindent 8 }}
   {{- end }}
 {{- end }}
+      {{- if .Values.imagePullSecret }}
+      imagePullSecrets:
+      - name: {{ .Values.imagePullSecret }}
+      {{- end }}
```

Comment:
nit: let's move this above `initContainers`

Reply:
fix: https://github.com/gravitational/teleport/pull/6941/commits/8e11fb6f669e3dc6e25e89436efc874c1146999e